persistence/4.0/_index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Jakarta Persistence 4.0"
+date: 2025-04-14
+summary: "Release for Jakarta EE 12"
+---
+Jakarta Persistence defines a standard for management of persistence
+and object/relational mapping in Java(R) environments.
+
+### New features, enhancements or additions
+
+- Introduce a new API exposing persistence operations for use with detached entities not associated with a persistence context, along with associated lifecycle callbacks
```

Comment:
For the record, I'm talking about `EntityAgent` here: https://github.com/jakartaee/persistence/issues/374